megamek/src/megamek/client/ui/displayWrappers/FontDisplay.java

```diff
@@ -35,6 +37,10 @@
     //endregion Variable Declarations
 
     //region Constructors
+    public FontDisplay(final String familyName) {
+        this(Font.decode(familyName).deriveFont(Font.PLAIN, 16f), familyName);
```

Comment:
If you're interested, you could try getting the UIManager default font for comboboxes and derive from it. The size would probably be 12 then.
Like:
Font font = UIManager.getFont("ComboBox.font");
if (font != null) etc.

![image](https://user-images.githubusercontent.com/17069663/155708776-201bd576-aff5-42b3-b9d0-3888ed2413b7.png)

Reply:
It's in a constructor, so it needs to be inline... which means using the UIManager isn't the most stable way to code. However, I've swapped to 12 as that's the default.